requirements/requirements.txt

```diff
@@ -0,0 +1,9 @@
+phoneme_guesser~=0.1
+mycroft-messagebus-client~=0.9.1,!=0.9.2,!=0.9.3
+inflection~=0.5
+pexpect~=4.8
+requests~=2.26
+json_database~=0.5
```

Comment:
Bump to 0.6ish if it tests working. This is an example of the Jarbas/Chance One-Digit-Off Semver Prerelease System in the wild 😘

Reply:
done.

Reply:
0.6 fixes a typo in the helper query class methods, bellow -> below, unused in ovos so far and safe to bump

it also reduces verbosity of logs which are super annoying when disabling Lock, this alone is worth the bump